src/10_functions.py

```diff
@@ -1,12 +1,22 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(num):
+    if num % 2 == 0:
+        return True
+    else:
+        return False
```

Comment:
Awesome. One thing we can do here is simply `return num % 2 == 0` which returns a boolean value.